viogpu/viogpudo/viogpudo.cpp

```diff
@@ -3411,6 +3411,8 @@ BOOLEAN VioGpuAdapter::CreateCursor(_In_ CONST DXGKARG_SETPOINTERSHAPE* pSetPoin
     UINT resid, format, size;
     VioGpuObj* obj;
     PAGED_CODE();
+    (void)pSetPointerShape; // unused
```

Comment:
Sorry, why those two are added in this commit?

Reply:
Because those variables are not used. They are only referenced with `DbgPrint` but when you build non-debug build where it's no-op you'll get compiler warning so by this we silence that warning.

Reply:
The correct way to do it is to use UNREFERENCED_PARAMETER macro

```
UNREFERENCED_PARAMETER(pSetPointerShape);
UNREFERENCED_PARAMETER(pCurrentMode);
```

Like here: 
https://github.com/virtio-win/kvm-guest-drivers-windows/blob/ec4a51758f1d53281c8a4d78956239ba74c2aaa4/vioscsi/virtio_pci.c#L123